plugins/routing/routing_plugin.js

```diff
@@ -702,10 +704,11 @@ class RoutingPlugin extends Plugin {
         }
         if (changeDesc) {
           if (changeActiveWanNeeded) {
+            this.processWANConnChange();
```

Comment:
Calling this here may be insufficient. Recommend calling it at the end of _applyActiveGlobalDefaultRouting